lib/pengine/unpack.c

```diff
@@ -89,14 +89,20 @@ pe_fence_node(pe_working_set_t * data_set, node_t * node, const char *reason)
         set_bit(node->details->remote_rsc->flags, pe_rsc_failed);
 
     } else if (is_baremetal_remote_node(node)) {
-        if(pe_can_fence(data_set, node)) {
-            crm_warn("Node %s will be fenced %s", node->details->uname, reason);
+        resource_t *rsc = node->details->remote_rsc;
+
+        if ((rsc != NULL) && (is_set(rsc->flags, pe_rsc_managed))) {
+            if(pe_can_fence(data_set, node)) {
```

Comment:
To the contrary, we want to avoid style-only commits. It's worthwhile to bring existing code to current style standards when it is being changed already (like here), but the time required for coding/review and the potential for accidentally introducing an error make large-scale style-only commits undesirable (the vast majority of the time -- there are exceptions when the code is difficult to work on without re-styling).